core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-reports/store/dot-experiments-reports-store.spec.ts

```diff
@@ -548,4 +523,188 @@ describe('DotExperimentsReportsStore', () => {
             });
         });
     });
+
+    describe('Show Bayesian Chart', () => {
+        it('should `hasEnoughDataForShowBayesianChart$` retrieve `false` when the BayesianResult have `NONE` as suggested winner', (done) => {
+            //The following mock has failures for both variants, enough sessions to display  results,
+            // so it generate the data sets; but no suggested winner
+            dotExperimentsService.getResults.mockReturnValue(
+                of({
+                    ...EXPERIMENT_MOCK_RESULTS,
+                    goals: {
+                        ...EXPERIMENT_MOCK_RESULTS.goals,
+                        primary: {
+                            ...EXPERIMENT_MOCK_RESULTS.goals.primary,
+                            variants: {
+                                ...EXPERIMENT_MOCK_RESULTS.goals.primary.variants,
+                                [DEFAULT_VARIANT_ID]: {
+                                    ...EXPERIMENT_MOCK_RESULTS.goals.primary.variants[
+                                        DEFAULT_VARIANT_ID
+                                    ],
+                                    uniqueBySession: {
+                                        count: 5,
+                                        totalPercentage: 100.0,
+                                        variantPercentage: 100.0
+                                    }
+                                },
+                                '111': {
+                                    ...EXPERIMENT_MOCK_RESULTS.goals.primary.variants['111'],
+                                    uniqueBySession: {
+                                        count: 8,
+                                        totalPercentage: 100.0,
+                                        variantPercentage: 100.0
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    sessions: { total: 20, variants: { DEFAULT: 10, '111': 10 } },
+                    bayesianResult: {
+                        ...EXPERIMENT_MOCK_RESULTS.bayesianResult,
+                        suggestedWinner: BayesianStatusResponse.NONE
+                    }
+                })
```

Comment:
I see that you are repeating this object through all this file and just changing one section. Should we do a builder function for this object?

```
getMockedResult({
                   count: 8,
                   totalPercentage: 100.0,
                   variantPercentage: 100.0
                });
```

Something like that.